index.d.ts

```diff
@@ -94,19 +94,25 @@ declare module "oncoprintjs"
         cell_height?: number;
         track_padding?: number;
         has_column_spacing?: boolean;
-        data_id_key?: string;
-        tooltipFn?:TrackTooltipFn<D>;
+        data_id_key?: keyof D;
```

Comment:
nice

Reply:
Doesn't actually do much so far though, as we are (as far as I have seen) just setting the type variable to `any` on the OncoprintJS instance level, rather than checking consistency within individual method invocations such as `.addTracks([trackParams])`. But at least it's more explicit in case we fix this at some point!